tslib/src/functions/gias-sync/index.ts

```diff
@@ -0,0 +1,20 @@
+import { AzureFunction, Context } from '@azure/functions'
+import { performance } from 'perf_hooks'
+const functionName = 'gias-sync'
+import { GiasService } from './gias.service'
+import config from '../../config'
+
+const timerTrigger: AzureFunction = async function (context: Context, timer: any): Promise<void> {
+  const start = performance.now()
+
+  const giasService = new GiasService()
+  const extractXml = await giasService.GetExtract(config.Gias.ExtractId)
+  context.bindings.giasExtractFile = extractXml
+
+  const end = performance.now()
+  const durationInMilliseconds = end - start
+  const timeStamp = new Date().toISOString()
+  context.log(`${functionName}: ${timeStamp} run complete: ${durationInMilliseconds} ms`)
```

Comment:
How fast do you expect this to run? ms / s / mm ?

Reply:
it usually takes between 3000 - 6000ms

The function is incomplete, as it only persists the extract.   Processing is yet to be implemented.  I will summarise this in the PR description.